ee/modules/650-runtime-audit-engine/images/falco/werf.inc.yaml

```diff
@@ -21,18 +22,31 @@ docker:
   CMD: ["/usr/bin/falco"]
 ---
 artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
-from: {{ $.Images.BASE_ALT_DEV }}
+from: {{ $.Images.BASE_ALT_P11 }}
 shell:
   install:
+  - apt-get update
+  - apt-get install -y git openssl-devel wget clang cmake libelf-devel libbpf-devel bpftool perl-IPC-Cmd gcc12 gcc12-c++ build-essential golang
```

Comment:
Have we changed strategy? And now we can install packages directly in the build file?
But what about assembly in a closed env?

Reply:
Yes, sorry for that. I've describe strategy. At general, in closed envs we will use apk packages proxy, this is tested and work. So we return to use general (not dev) images and use packages installer because that more flexible way.